### PR TITLE
Updated to upstream version 0.14.7 and fix EL7 build

### PR DESCRIPTION
**Pull request by a15y87**

### CODE REVIEW

**Comment by a15y87** [comment]
%define fluentd_log_dir         %{_localstatedir}/log/fluentd
%define fluentd_buffer_dir 	%{_localstatedir}/spool/fluentd/buffer
%define fluentd_pos_dir         %{_localstatedir}/spool/fluentd/pos
--
fluentd_log_dir - logs dir
fluentd_buffer_dir - temporary buffer
fluentd_pos_dir - position files dir

File: fluentd.spec [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
-BuildRequires:  rubygem(test-unit-rr)
-BuildRequires:  rubygem(yajl-ruby)
-BuildRequires:  rubygem(msgpack)
-BuildRequires:  rubygem(sigdump)
-BuildRequires:  rubygem(cool.io)
-BuildRequires:  rubygem(tzinfo)
-BuildRequires:  rubygem(tzinfo-data)
-BuildRequires:  rubygem(http_parser.rb)
-BuildRequires:  rubygem(thread_safe)
+BuildRequires:  rubygem-test-unit
```
Comment (mrunge) [inline fluentd.spec]: why changing the package requirements here?

Comment (paramite) [inline fluentd.spec]: Yes exactly. What is the reason for removing virtual requirements?

Comment (a15y87) [inline fluentd.spec]: rpm generators on EL7 does not automatically create virtual requirements:

rpm -qp --provides rubygem-strptime-0.1.8-1.el7.centos.x86_64.rpm 
rubygem-strptime = 0.1.8-1.el7.centos
rubygem-strptime(x86-64) = 0.1.8-1.el7.centos
---
rpm -qp --provides rubygem-strptime-0.1.8-1.fc24.x86_64.rpm 
rubygem(strptime) = 0.1.8-1
rubygem-strptime = 0.1.8-1.fc24
rubygem-strptime(x86-64) = 0.1.8-1.fc24

**Pull request closed without merging**